scripts/run_frontend_tests.sh

```diff
@@ -37,28 +37,60 @@ set -e
 source $(dirname $0)/setup.sh || exit 1
 source $(dirname $0)/setup_gae.sh || exit 1
 
-# Install third party dependencies
-# TODO(sll): Make this work with fewer third-party dependencies.
-bash scripts/install_third_party.sh
-
-# Ensure that generated JS and CSS files are in place before running the tests.
-echo ""
-echo "  Running build task with concatenation only "
-echo ""
+# Parse additional command line arguments.
+# Credit: http://stackoverflow.com/questions/192249
+SKIP_INSTALL=false
```

Comment:
Suggest SKIP_INSTALLING_THIRD_PARTY_LIBS

('install' seems too general -- to me, it refers to setting up Oppia itself as well)